docs/integrators/advanced-observer-settings.md

```diff
@@ -0,0 +1,89 @@
+---
+id: advanced-observer-settings
+title: Advanced Observer Settings
+---
+
+[comment]: # (mx-abstract)
+
+This page describes some of the settings an integrator might want to apply on the observers in order to better make use of the nodes.
+
+For the settings from the `config.toml` file that are needed to be altered, we recommend using the `OverridableConfigTomlValues` section found in the `prefs.toml` file. 
```

Comment:
Optional - add link to:

https://docs.multiversx.com/validators/node-configuration/#overriding-configtoml-values

Reply:
good point, will fix